### DISCUSSION TITLE
Open API Block - add option to default to Curl instead of Javascript

### DISCUSSION THREAD

**Discussion by brianshih1** [GitbookIO/community#790] [Feature Requests]
For the OpenAPI block, I would like the option to choose to default to `Curl` instead of `Javascript`? Also, it would be awesome if I can hide certain languages (e.g. only have `Curl`).

<img width="513" alt="Screenshot 2024-11-08 at 10 07 42 PM" src="https://github.com/user-attachments/assets/5153b303-149d-4f77-a1d9-0da9038e9ee9">

**Comment by aikoii** [comment]
👋 @brianshih1! You can override this by providing your own code samples, which can be in any order or format you prefer. Here’s a guide that explains how to do this: https://docs.gitbook.com/content-editor/blocks/openapi#options
